src/openads/infrastructure/position/positionCreatedObserver.js

```diff
@@ -8,7 +8,7 @@ const positionCreatedObserverFactory = appnexusConnector => appNexusConsumersRep
       invCode: payload.placement,
       sizes: payload.sizes,
       keywords: payload.segmentation,
-      native: payload.native.fields
+      native: payload.native ? payload.native.fields : undefined
```

Comment:
Well, I knew that this day will came sooner or later, in es6 there is a crazy thing like this:
https://stackoverflow.com/questions/12878612/assignment-with-double-ampersand

I really don't understand in a clear way the explanation but seems to work like a optional...
https://jsbin.com/tavobevuhe/edit?js,console,output

Reply:
I've seen that in use for conditional rendering in react... Don't know if it's clearer than the ternary operator ...

Reply:
Done!

![](https://media.giphy.com/media/8BmvqIB4PWafm/giphy.gif)